pkg/dump/dump.go

```diff
@@ -836,6 +878,94 @@ func GetAllConsumerGroups(ctx context.Context,
 	return consumerGroupObjects, nil
 }
 
+// GetAllConsumerGroupsDefault queries Kong for all the ConsumerGroups using client.
+// This does not include "consumer-group-plugins" or policy overrides associated
+// with a consumer-group
+func GetAllConsumerGroupsDefault(ctx context.Context,
```

Comment:
I added new functions rather than changing the func signature or behaviour here, as these are exposed and may be used in other external modules, other than deck.